packages/build-utils/test/unit.test.ts

```diff
@@ -713,6 +713,43 @@ it('should detect package.json in nested frontend', async () => {
   expect(result.packageJsonPath).toEqual(path.join(fixture, 'package.json'));
 });
 
+it('should detect `packageManager` in npm monorepo', async () => {
+  try {
+    process.env.ENABLE_EXPERIMENTAL_COREPACK = '1';
+
+    const base = path.join(__dirname, 'fixtures', '41-npm-workspaces-corepack');
+    const fixture = path.join(base, 'a');
+    const result = await scanParentDirs(fixture, false, base);
+    console.log(result);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏

```suggestion
```